src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
Okay after some discussion with @lynyus and after checking the code again, I would recommend to remove this getter completely as it does not provide any benefit. Just directly use `getField(FieldName.TITLE)` in the code.